scss/main.scss

```diff
@@ -1,150 +1,120 @@
 @import 'scss/variables';
-@import 'node_modules/bootstrap-sass/assets/stylesheets/bootstrap';
+@import 'node_modules/bootstrap/scss/bootstrap';
 
+// flush footer to bottom
+html {
+    height: 100%;
+}
 
 body {
+    min-height: 100%;
     position: relative;
+    padding-bottom: 40px;
 }
 
-pre {
-    margin-bottom: 20px;
+footer {
+    // scss-lint:disable ImportantRule
+    position: absolute  !important;
+    right: 0;
+    bottom: 0;
+    left: 0;
 }
 
 
-code {
-    // Allow wrapping of code
-    white-space: normal;
+// additional spacing for sticky object
+.sticky-top {
+    top: $spacer * 3;
 }
 
-.nowrap {
-    white-space: nowrap;
-}
 
+// show anchor on hover
 h1,
-.page-header:first-child {
-    margin-top: 0;
-}
-
-// Navbar
-
-.pc-navbar-brand {
-    padding: 13px 15px;
-}
-
-.logo {
-    height: 44px;
-    width: 44px;
-    display: block;
-    background: url('../logo.png');
-}
-
-.pc-navbar-collapse {
-    @include box-shadow(none);
-}
-
-.pc-navbar .btn {
-    margin: 18px 15px;
-    padding: 6px 12px;
-}
-
-
-// Top shelf
-
-.top-shelf {
-    margin-bottom: 40px;
-    background: $pc-green-light;
-    color: white;
-
-    .container {
-        position: relative;
+h2,
+h3 {
+    small {
+        display: none;
     }
 
-    .pc-logo-top-shelf {
-        float: right;
-        margin: -30px 20px 0 40px;
+    &:hover small {
+        display: inline;
     }
+}
 
-    h1 {
-        font-weight: bold;
-        margin: 40px 0 20px;
 
-        a {
-            color: $pc-yellow;
+// underline headline
+.page-header {
```

Comment:
It looks like the suggested way to do this is with the helper classes?

https://stackoverflow.com/a/49708022/812183

Reply:
mhm yes i can do it that way, but imo the custom class might be cleaner (and more understandable) in the html (and as we are using scss, the border color etc. is inherited from the theme)

Reply:
Ah ok cool, sounds good